umbraco/Umbraco-CMS
# improve missingProperties data returned for missing propertie values

### Description
Missing properties extension of the ContentControllerBase was missing some information to properly identify the property in question in a variant context.

### Notes
The ReponseModel structure was validated with the front-end.

### Testing
- Create a document with some required properties
- Save a document of that type with some/all of the required properties not filled in.
- Validate the document trough the management api
- The response should have a value named "missingValues" with clear information about which property in which context is empty.
